core/bin/external_node/src/main.rs

```diff
@@ -139,6 +139,7 @@ async fn run_tree(
             .merkle_tree_include_indices_and_filters_in_block_cache,
         memtable_capacity: config.optional.merkle_tree_memtable_capacity(),
         stalled_writes_timeout: config.optional.merkle_tree_stalled_writes_timeout(),
+        sealed_batches_have_protective_reads: config.optional.protective_reads_persistence_enabled,
```

Comment:
I am not 100% sure about this. My thinking was:
case 1) if EN is not persisting protective reads then it is also not running tree
case 2) if EN is persisting protective reads then we can run metadata calculator with `sealed_batches_have_protective_reads=true`

@perekopskiy could you double check this please?

Reply:
Yes, correct. More precisely -- EN never runs full tree, so it doesn't matter what we set in `sealed_batches_have_protective_reads`